catkin_ws/src/propulsion/src/drytest.py

```diff
@@ -94,9 +94,12 @@ def re_arm():
         forwards_test(7)
         forwards_test(8)
     elif choice == "2":
-        choice = int(input("select thruster to test (1-8): "))
-        if 1 <= choice and choice <= 8:
-            forwards_test(choice)
+        while True:
+            choice = int(input("select thruster to test (1-8): "))
+            if 1 <= choice and choice <= 8:
+                forwards_test(choice)
```

Comment:
I think you can simplify it so that it doesn't use the forward_test function. Maybe just run whatever number is pressed (e.g., if you want to run thruster 7 twice and then thruster 8 twice, the input would be 7788 instead of 712812). This way, you could fully remove the first if statement (if choice == "1") in the main while loop - redundant code.

Reply:
ok ill test tn w new version